app/src/main/java/com/kickstarter/libs/utils/ViewUtils.java

```diff
@@ -32,6 +36,12 @@
 public final class ViewUtils {
   private ViewUtils() {}
 
+  public static void addBoldSpan(final @NonNull SpannableString spannableString, final @NonNull String substring) {
+    final int index = spannableString.toString().indexOf(substring);
```

Comment:
What happens if the `substring` can't be found in the `spannableString`? 🤔

Reply:
i let it blow up, (which happened yesterday because I used the wrong string 😛) I can abandon ship if the string can't be found!